cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -108,13 +111,14 @@ private static String findNetwork(Compute compute, String project) throws IOExce
       NetworkList networkList = compute.networks().list(project).execute();
       networks = networkList.getItems();
     } catch (Exception e) {
-      handleRetryableExceptions(e);
-      throw e;
+      handleRetryableExceptions(e, ErrorTagProvider.ERRGP_CONFIGURATION);
+      throw new DataprocRuntimeException(e, ErrorTagProvider.ERRGP_CONFIGURATION, ErrorTagProvider.ERRCD_NETWORK);
```

Comment:
Usually it's either `handleBlah` or `throw handleBlah`

Reply:
handleRetryableException was already there.